cfme/tests/infrastructure/test_host.py

```diff
@@ -92,6 +92,8 @@ def _check_items_visibility():
 
 @pytest.mark.rhv2
 # Tests to automate BZ 1201092
+@pytest.mark.meta(blockers=[BZ(1619626, forced_streams=['5.8'],
```

Comment:
**Question:**
are you sure, you want to block it for 5.8? I found this raised against 5.9  and no comment about 5.8.

Reply:
Hello @digitronik , right, that's a good point. I should create a BZ clone for 5.8 first. Moving this to WIP for now.